src/main/io/gps.c

```diff
@@ -998,7 +1016,8 @@ void gpsConfigureNmea(void)
             } else if (gpsData.state_position < 2) {
                 // *** this message also appears to fail ***//
                 // NMEA reports back at whatever speed the module is configured to send at, not 5Hz
-                serialPrint(gpsPort, "$PSRF103,00,6,00,0*23\r\n"); // set GGA rate to 5Hz
+                serialPrint(gpsPort, "$PSRF103,00,00,01,01*25\r\n"); // set GGA rate to 1Hz
+                // serialPrint(gpsPort, "$PSRF103,00,00,05,01*21\r\n"); // set GGA rate to 5Hz
```

Comment:
(Same as other comments on using SiRF Tech proprietary commands.)
My understanding is the proprietary commands are for specific manufacturers (u-blox = $PUBX, SiRF Tech = $SIRF, GlobalTop Tech = $PMTK).
Would specific modules respond to other manufacturer's proprietary commands?

Reply:
SiRF is a binary protocol, the PSRF103 commands can also be used to set NMEA message type.

Reply:
The SiRF NMEA commands we are using above are ASCII ($PSRF...), not binary, right?

Reply:
Only fixed the checksum here.